src/java/org/apache/cassandra/index/sai/disk/hnsw/VectorPostingsWriter.java

```diff
@@ -24,12 +24,30 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Function;
+
+import com.google.common.collect.BiMap;
 
 import org.apache.cassandra.io.util.SequentialWriter;
 import org.apache.cassandra.utils.Pair;
 
 public class VectorPostingsWriter<T>
 {
+
+    private final boolean useOrdinalsMapper;
+
+    private final Function<Integer, Integer> reverseOrdinalsMapper;
```

Comment:
nit: not sure what "reverse" adds to the name here, wouldn't `ordinalsMapper` be as meaningful but shorter?

Reply:
this is to indicate that newOrdinals.inverse() is used (and to match HnswGraphWriter's)